### DISCUSSION TITLE
RSS文件混杂单集和多集的情况

### DISCUSSION THREAD

**Discussion by beiyanpiki** [wushuo894/ani-rss#88] [Q&A]
作者您好，我使用软件时有个问题，有些字幕组会将多集合并成一个种子发布，如下图，这样会导致单集、多集混杂，在ani-rss里就会出现这些多集的视频缺失，这种情况应该如何配置。

![image](https://github.com/user-attachments/assets/ff48dcba-6b9b-4694-8d65-a34f96cbfd41)

![image](https://github.com/user-attachments/assets/71d2277c-2dec-4259-bdcf-09ddd35c031d)

自定义集数规则能否解决这个问题，我看文档不是写的很明白，自定义集数后的数字选择框是什么含义？

**Comment by wushuo894** [comment]
暂时不支持合集
自定义集数后的数字是正则匹配到的组序号

**Reply by beiyanpiki** [reply]
后续会有计划支持多集内容吗

**Reply by wushuo894** [reply]
改动应该很多，暂时没有计划

你找找没有合集的rss吧

**Reply by beiyanpiki** [reply]
好的 谢谢
